contents/docs/libraries/nuxt-js.md

```diff
@@ -99,3 +167,9 @@ Let's say for example the user makes a purchase you could track an event like th
 ...
 </script>
 ```
+
+## Further reading
```

Comment:
I think it is ok to leave this out in docs.

Reply:
I think it's helpful to give visibility to related tutorials